commcare_connect/opportunity/views.py

```diff
@@ -420,14 +420,23 @@ def review_visit_export(request, org_slug, pk):
 def export_status(request, org_slug, task_id):
     task_meta = AsyncResult(task_id)._get_task_meta()
     status = task_meta.get("status")
+    print(status)
     progress = {
         "complete": status == "SUCCESS",
     }
     if status == "FAILURE":
         progress["error"] = task_meta.get("result")
+
+    template = (
+        "tailwind/components/upload_progress_bar.html"
+        if request.GET.get("ui") == "new"
```

Comment:
Let's not mix old and new views like this, we better just create a separate view for the new UI.